bolt/bolt-protocol-message-specification-4.md

```diff
@@ -500,11 +500,11 @@ extra::Dictionary(
 
   For **Auto-commit Transaction** (`RUN`) the `extra` field carries:
 
-  - The `bookmarks` is a list of strings containg some kind of bookmark identification e.g ["neo4j-bookmark-transaction:1", "neo4j-bookmark-transaction:2"]
-  - The `tx_timeout` is an integer in that specifies a transaction timeout in ms.
-  - The `tx_metadata` is a dictionary that can contain some metadata information, mainly used for logging.
-  - The `mode` specifies what kind of server the `RUN` message is targeting. For write access use `"w"` and for read access use `"r"`. Defaults to write access if no mode is sent.
-  - The `db` specifies the database name for multi-database to select where the transaction takes place. If no `db` is sent or empty string it implies that it is the default database.
+  - `bookmarks`: it is a list of strings containing some kind of bookmark identification e.g. ["neo4j-bookmark-transaction:1", "neo4j-bookmark-transaction:2"]
+  - `tx_timeout`: it is an integer in that specifies a transaction timeout in ms.
+  - `tx_metadata`: it is a dictionary that can contain some metadata information, mainly used for logging.
+  - `mode`: it specifies what kind of server the `RUN` message is targeting. For write access use `"w"` and for read access use `"r"`. Defaults to write access if no mode is sent.
+  - `db`: it specifies the database name for multi-database to select where the transaction takes place. If no `db` is sent or empty string it implies that it is the default database.
```

Comment:
I think removing "it is" or "it" at the beginning of each item completely is even more concise.